README.md

```diff
@@ -1,486 +1,679 @@
+        set.call(this, value);
+        this.render();
+      }
+    }
+  };
+}
 ```
 
-### `@tracked`
-
-The `@tracked` decorator turns a public field declaration into a getter/setter pair which triggers a `render()` method when the setter is called. This pattern, or patterns like it, is common in frameworks to avoid extra bookkeeping scattered throughout the application to ask for re-rendering.
+This example could be roughly desugared as follows:
 
 ```mjs
-import { @tracked } from "./tracked.mjs";
+let initialize, get, set;
 
 class Element {
-  @tracked counter = 0;
+  #counter = initialize(0);
+  get counter() { return this.#counter; }
+  set counter(v) { this.#counter = v; }
 
   increment() { this.counter++; }
 
   render() { console.log(counter); }
 }
 
-const e = new Element();
-e.increment();  // logs 1
-e.increment();  // logs 2
+{ get, set } = Object.getOwnPropertyDescriptor(Element.prototype, "counter");
+{ get, set, initialize } = tracked({get, set}, { kind: "field", name: "counter", isStatic: false })
+Object.defineProperty(Element.prototype, "counter", {get, set});
 ```
 
-`@tracked` is defined in terms of the combination of two built-in decorators that we've seen before above. `@initialize` is used to replace the property definition with setting a property which stores the underlying data. `@register` is used to define a getter/setter pair which is used when accessing the property.
+### Limited access to private fields and methods
 
-```mjs
-// tracked.mjs
+Sometimes, certain code outside of a class may need to access private fields and methods. For example, two classes may be "collaborating", or test code in a different file needs to reach inside a class.
 
-export decorator @tracked {
-  @initialize(function(value, name) { this[`__internal_${name}`] = value; })
-  @register((target, name) => {
-    Object.defineProperty(target, "name", {
-      get() { return this[`__internal_${name}`]; },
-      set() { this[`__internal_${name}`] = value; this.render(); },
-      configurable: true
-    });
-  })
-}
-```
-
-Note, further built-in decorators as in [NEXTBUILTINS.md](./NEXTBUILTINS.md#tracked) may provide a more direct and statically analyzable way to implement `@tracked` and avoid the use of `Object.defineProperty`.
-
-### `@bound`
-
-The `@bound` decorator makes a method auto-bound: it will carry around the original `this` value when accessed as `this.method` and not immediately called. This behavior matches Python's semantics, and it's been found useful in the React ecosystem, which makes frequent use of passing functions around. Example usage:
+Decorators can make this possible by giving someone access to a private field or method. This may be encapsulated in a "private key"--an object which contains these references, to be shared only with who's appropriate.
 
 ```mjs
-import { @bound } from "./bound.mjs";
+import { PrivateKey } from "./private-key.mjs"
 
-class Foo {
-  x = 1;
+let key = new PrivateKey;
```

Comment:
```suggestion
let key = new PrivateKey();
```

Reply:
This is a stylistic preference; I don't see a reason to make the change.